libnavigation-util/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="mapbox_unit_kilometers">km</string>
+    <string name="mapbox_unit_meters">m</string>
+    <string name="mapbox_unit_miles">mi</string>
+    <string name="mapbox_unit_feet">ft</string>
+    <string name="mapbox_unit_yards">yd</string>
```

Comment:
Po polsku mamy:
- kilometry
- mile
- stopy
- jardy

Idk what abbreviations we can use for thise.

Reply:
These are translations of abbreviations which wouldn't fit in trip summary/notification as long form. Looks like English short forms seem to be universally accepted in UI elements for these purposes.

Reply:
We might need to use string-plurals for this:
1 kilometr, 2 kilometry, 10 kilometrow
1 metr, 2 metry, 10 metrow
1 mila, 2 mile, 10 mil
1 stopa, 2 stopy, 10 stop
1 jard, 2 jardy, 10 jardow